yarn.lock

```diff
@@ -1019,6 +1089,13 @@
   dependencies:
     "@babel/helper-plugin-utils" "^7.10.4"
 
+"@babel/plugin-syntax-top-level-await@^7.8.3":
```

Comment:
Can we dedupe this? There are a few duplicates floating around now. plugin-syntax-typescript now has 3 versions.

Reply:
plugin-syntax-typescript is down to two now

so without manually doing a bunch of things, I'm not sure how much deduping you want done

a better approach might be to open a new branch after this one where we either run yarn deduplicate or we wipe out our yarn lock and let yarn take care of everything